cubelib/src/algs.rs

```diff
@@ -83,6 +83,15 @@ impl Algorithm {
     }
 }
 
+impl InvertibleMut for Algorithm {
```

Comment:
Not sure if this behaves exactly as expected. Could move this into the `invert` sub-command